source/linear-algebra/source/02-EV/03.ptx

```diff
@@ -288,6 +363,137 @@ we'll learn what this means more carefully in a later chapter.
 		</figure>
 </observation>
 
+  <activity xml:id="EV3-invalid-proof">
+      <statement>
+        <p> 
+            To show that sets of Euclidean vectors form subspaces, we will
+            need to prove that certain equalities hold.
+        </p>
+        <p>
+            Consider the following argument that 
+            <m>5=7</m>:
+<md>
+    <mrow>
+&amp;&amp;    5&amp;=7
+    </mrow>
+    <mrow>
+ &amp;  \Rightarrow&amp;  5-6&amp;=7-6
+    </mrow>
+    <mrow>
+ &amp;  \Rightarrow&amp;  -1&amp;=1
+    </mrow>
+    <mrow>
+ &amp;  \Rightarrow&amp;  (-1)^2&amp;=(1)^2
+    </mrow>
+    <mrow>
+ &amp;  \Rightarrow&amp;  1&amp;=1
+    </mrow>
+</md>
+            </p>
+        <p>
+            Is this reasoning valid?
```

Comment:
[Activity 2.3.9](https://b4d2c2b0.tbil.pages.dev/preview/linear-algebra/instructor/EV3#EV3-invalid-proof) 
https://b4d2c2b0.tbil.pages.dev/preview/linear-algebra/instructor/EV3#EV3-invalid-proof

Demonstrate an invalid proof earlier, to discourage such proofs in following activities.

Reply:
I kept the original invalid proof as a flexible extension, at the end of the section, to reinforce the issue.